gcloud-java-storage/src/main/java/com/google/gcloud/spi/DefaultStorageRpc.java

```diff
@@ -521,4 +522,31 @@ public String open(StorageObject object, Map<Option, ?> options)
       throw translate(ex);
     }
   }
+
+  @Override
+  public RewriteResponse rewrite(StorageObject source, Map<Option, ?> sourceOptions,
+      StorageObject target, Map<Option, ?> targetOptions, String token, Long maxByteRewrittenPerCall)
+      throws StorageException {
+    try {
+      return storage
+          .objects()
+          .rewrite(source.getBucket(), source.getName(), target.getBucket(), target.getName(),
+              target)
+          .setRewriteToken(token)
+          .setMaxBytesRewrittenPerCall(maxByteRewrittenPerCall)
```

Comment:
I assume `null` is fine?

Also, since it seems that this value ["must be an integral multiple of 1 MiB (1048576)"](https://developers.google.com/resources/api-libraries/documentation/storage/v1/java/latest/com/google/api/services/storage/Storage.Objects.Rewrite.html#getMaxBytesRewrittenPerCall()) I think it would be nicer the API accepts it in MB (and the rpc layer will multiple the value by 1048576).